target/linux/ath79/image/generic.mk

```diff
@@ -1745,24 +1748,24 @@ define Device/netgear_ex6400_ex7300
   IMAGE/factory.img := $$(IMAGE/default) | netgear-dni | check-size
   DEVICE_PACKAGES := kmod-ath10k-ct ath10k-firmware-qca99x0-ct
```

Comment:
Please add:
````
SUPPORTED_DEVICES += netgear_ex6400
````
This will add a tag to the image which tells swupgrade that the new `netgear_ex7300` image can be installed over an image with the name `netgear_ex6400` without problems. 

Without this change swupgrade will reject the new firmware when you want to upgrade it over an existing image.

Please try if this really work too.

Reply:
Done. I had to use a slightly different line `SUPPORTED_DEVICES += netgear,ex6400` and I confirmed that sysupgrade/LuCI does not complain when upgrading.